### PR TITLE
Feature: Out of the box min

### PR DESCRIPTION
**Pull request by tiagomapmarques**
When the package is published in npm we can use it out-of-the-box with unpkg for examples (although this would be discouraged).
Note: `min` file is fully transpiled to es5

Example usage:
```html
<!DOCTYPE html>
<html lang="en">
<head>
  <title>Biotope Element</title>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/2.2.10/webcomponents-bundle.js"></script>
  <script src="https://unpkg.com/@biotope/element/lib/biotope-element.min.js"></script>
</head>
<body>
  <my-button greet="hello world"></my-button>

  <script>
    class MyButton extends BiotopeElement {
      render() {
        return this.html`<h1>${this.props.greet}</h1>`;
      }
    }
    MyButton.attributes = ['greet'];
    MyButton.register();
  </script>
</body>
</html>
```

### CODE REVIEW

File: src/min.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* istanbul ignore next */
+import Component from '.';
+
+/* istanbul ignore next */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+(window as any).BiotopeElement = Component;
```
Comment (tiagomapmarques) [inline src/min.ts]: Discussion:
If this is approved, which name would be best? `Component`, `Element`, `BiotopeElement`? Something else?

Comment (timomayer) [reply src/min.ts]: BioElement?

**Comment by jurekbarth** [comment]
@tiagomapmarques why is that necessary? You can simply import the esm version, e.g.:
```
<my-button greet="hello world"></my-button>
<script type="module">
  import Component from "https://unpkg.com/@biotope/element@4.0.0-4/lib/esm/index.js";

  class MyButton extends Component {
    render() {
      return this.html`<h1>${this.props.greet}</h1>`;
    }
  }
  MyButton.attributes = ["greet"];
  MyButton.register();
</script>
```

**Comment by tiagomapmarques** [comment]
@jurekbarth we can yes. But that's for modern browsers only. And it's non-minified. The ES5 version is more versatile.

File: src/min.ts (hunk 1 above)
Comment (SheepFromHeaven) [reply src/min.ts]: Voting for `BiotopeElement` as it is the most explicit one

**Comment by jurekbarth** [comment]
@tiagomapmarques i agree if we want to support IE11 then you need a compiled ES5 version, but here your code from above will also not work. That's why i still don't get the use case. 
A minified version makes sense for me, if we would encourage to use it in that way. As this is discouraged there's no point for minified version.

**Comment by tiagomapmarques** [comment]
My example above was just to make it easier for the eyes lol I didn't want to write full ES5 as it's really ugly.
When I said "highly discouraged", I meant adding the lib to a project via CDN. We should always strive for a build system that fits our needs. But some projects may have this constraint. This is the reason behind this `min` version. It can be served optimally, with no additional work needed, when you have the need to exclude it from your build.
Some examples:
1) you still use bower and don't want to over-complicate your build
2) you just need a html file, not a full project
3) you're using some online editor which transpiles your code but not libs and want to try out `biotope-element` on IE11
...

**Pull request merged**